index.html

```diff
@@ -49,7 +49,6 @@ <h1>Im Lucca Glad to see you</h1>
         <h5>Lets Connect</h5>
```

Comment:
`footer` tag is reserved for the footer of the page. It would be wise to use a `footer` tag in the heading of the page itself. I recommend you change it with a `div`.

Reply:
Implemented 👍